test/services/SubmissionServiceTest.scala

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import base.UnitSpec
+import connectors.CustomsDeclareExportsMovementsConnector
+import connectors.exchanges.{Consolidation, DisassociateDUCRRequest}
+import metrics.MovementsMetrics
+import models.ReturnToStartException
+import models.cache.DisassociateUcrAnswers
+import org.mockito.ArgumentCaptor
+import org.mockito.ArgumentMatchers._
+import org.mockito.BDDMockito._
+import org.mockito.Mockito._
+import org.scalatest.BeforeAndAfterEach
+import play.api.test.Helpers._
+import repositories.MovementRepository
+import services.audit.AuditService
+import uk.gov.hmrc.http.HeaderCarrier
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+class SubmissionServiceTest extends UnitSpec with BeforeAndAfterEach {
```

Comment:
Can we use `Spec` instead of `Test` to be consistent with others tests

Reply:
IntelliJ being too clever again...